app/js/component/data/github_issues.js

```diff
@@ -325,9 +329,16 @@ function (defineComponent, withAuthTokeFromHash, repositoriesURLs, withPopoverTe
             });
           }
 
+          var that = this;
           $.ajax({
             type: 'DELETE',
-            url: url + "/labels/" + oldLabel + this.getAccessTokenParam()
+            url: url + "/labels/" + oldLabel + this.getAccessTokenParam(),
+            error: function() {
+              $('#closeModalBtn').click(function(){
+                location.reload(true);
```

Comment:
Why is the page reloaded here? The issue should just go back to the original column. Is there no way to do this without reloading?